streamz/tests/test_kafka.py

```diff
@@ -223,6 +223,7 @@ def test_kafka_batch():
         stream.upstream.stopped = True
 
 
+@flaky(max_runs=3, min_passes=1)
```

Comment:
Any idea why? We would rather not have this. How frequently does it fail?

Reply:
Nope, no idea at all. This was really surprising for me, too. Unlike other flaky tests, this one passes >90% of the time, but I did see it fail one time, and succeed the next. Hence, I thought adding `flaky` would be best. Should I remove it?

Reply:
Do you believe it was already failing sometimes before, or did something change?

Reply:
Maybe it was a one-off, although it is concerning.

Reply:
I don't think anything changed, but I did recall Jarod saying that it failed for him once too, and this was way before the checkpointing stuff.

Reply:
I would be tempted to leave it as was, and see how often this arises. We can investigate at a later date whats going on - I don't suppose we even know at what stage things are hanging.

Reply:
Gotcha, will remove the flaky marker then.

Reply:
Actually, I'm running all tests locally, and this test seems to fail like 1 out of 5 times.